baseline/metadata.yaml

```
@@ -0,0 +1,50 @@
+metadata:
+  id: OSPS-B
+  title: Open Source Project Security Baseline
+  description: |
+    The Open Source Project Security (OSPS) Baseline is a set of security
+    criteria that projects should meet to demonstrate a strong security posture.
+  applicability-categories:
+    - id: Maturity Level 1
```

Constrain the identifiers to strings without spaces. These will get composed and fed into other systems that could may not handle multiple words
```suggestion
    - id: MA-1
```